pkg/operator/ceph/controller/spec.go

```diff
@@ -692,6 +692,13 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 			size = 1
 			logger.Info("maxLogSize is 0M setting to minimum of 1M")
 		}
+
+		if strings.Contains(daemonID, "*-client.rbd-mirror*") && size < 1000 {
+			size = 1000
```

Comment:
Can the number of rotations be bumped as well?

Reply:
rotation 7 is what default value is, @travisn thoughts ^^

Reply:
> rotation 7 is what default value is

We are changing the default `maxsize`, should be able to change the default `rotate` too?

Reply:
we can change the rotation too but the question is do we need to? I'm asking since the rotation of 7 is there from start and `maxlogSize` is something we had to introduce in order to stop filling the disk in case of huge logs

Reply:
I'm referring you back to https://bugzilla.redhat.com/show_bug.cgi?id=2159676#c3:
```
-rw-r--r--. 1  167  167  16M Jan  4 17:37 ceph-client.rbd-mirror.a.log.1.gz
-rw-r--r--. 1  167  167  16M Jan  4 16:37 ceph-client.rbd-mirror.a.log.2.gz
-rw-r--r--. 1  167  167  19M Jan  4 15:52 ceph-client.rbd-mirror.a.log.3.gz
-rw-r--r--. 1  167  167  20M Jan  4 14:52 ceph-client.rbd-mirror.a.log.4.gz
-rw-r--r--. 1  167  167  22M Jan  4 12:22 ceph-client.rbd-mirror.a.log.5.gz
-rw-r--r--. 1  167  167  15M Jan  4 00:07 ceph-client.rbd-mirror.a.log.6.gz
-rw-r--r--. 1  167  167  18M Jan  3 16:37 ceph-client.rbd-mirror.a.log.7.gz
```
The full rotate set took less than 200M there.  This is insufficient and needs to be bumped, 4x at least.  Since `maxsize` got doubled, I'm suggesting that we double `rotation` as well to get to 4x in total.  Or perhaps we could bump just `rotate`?

Bumping just `maxsize` (i.e. setting it to 2G) is less preferable since the uncompressed file would indeed be on the bigger side, compression would take longer, etc.  Keeping `maxsize` smaller but allowing a more rotations is nicer IMO.

Reply:
@travisn thoughts ^^

Reply:
A few thoughts:
- Since the need to increase the default logging size is temporary, I agree that a temporary customization just for the rbd mirror daemon sounds reasonable. This allows us to stay away from more custom settings in the CRD that would be daemon-specific.
- The rbd mirror daemon is not enabled by default and not commonly used, so increasing the log collection is not going to affect many clusters
- Good point about bigger files taking longer to rotate, so if we just set the rotation to 4x instead of the default of 7, that seems simple enough and better to spread out the perf impact of rotation. Then no need to increase the max log size in this case.

Reply:
so if I understand correctly, you suggesting to just increasing the rotation from 7 to 4x i.e 28 and not increasing the maxLogSize. From the perspective of the change, this will be more work compared to just increasing the maxLogSize as we'll require to pass another variable `rotation` to the script from all the ceph daemon container

Reply:
Yes that's the suggestion. To keep the api simpler, perhaps a wrapper method could just pass the default of 7 so that the mirror daemon is the only one that would need to call a different method with 28.

Reply:
done